addons/test/addonsBundle.spec

```diff
@@ -0,0 +1,26 @@
+const addonsBundle = (isVerify()) ?
+  require('../dist/textMaskAddons.js') :
+  require('../src/index.js')
+
```

Comment:
I guess the `index.js` file is in the wrong place.

Also could you add the `.js` suffix to this file `addons/test/addonsBundle.spec.js`